core/src/test/java/com/yahoo/oak/MultiThreadTest.java

```diff
@@ -36,29 +47,24 @@ public void init() {
                 .setChunkMaxItems(MAX_ITEMS_PER_CHUNK);
         oak = builder.build();
         latch = new CountDownLatch(1);
-        threads = new ArrayList<>(NUM_THREADS);
+        executor = Executors.newFixedThreadPool(NUM_THREADS);
     }
 
     @After
     public void finish() {
         oak.close();
     }
 
-    class RunThreads implements Runnable {
+    class RunThreads implements Callable {
         CountDownLatch latch;
 
         RunThreads(CountDownLatch latch) {
             this.latch = latch;
         }
 
         @Override
-        public void run() {
-            try {
-                latch.await();
-            } catch (InterruptedException e) {
-                e.printStackTrace();
-            }
-
+        public Void call() throws InterruptedException {
+            latch.await();
```

Comment:
This was indeed incorrect to ignore this exception.
Can you remove the try/catch clause in the other tests you modify as well (e.g., `RunThreadsDescend`).